src/main/java/bookface/model/person/Person.java

```diff
@@ -60,17 +48,24 @@ public Email getEmail() {
         return email;
     }
 
+    public boolean hasBooksOnLoan() {
+        return loanedBooks.size() > 0;
+    }
 
-    public Set<Title> getLoanedTitlesSet() {
+    public Set<Book> getLoanedBooksSet() {
         return loanedBooks;
     }
 
-    public String getLoanedTitlesDisplayString() {
+    public String getLoanedBooksDisplayString() {
         return loanedBooks.toString();
     }
 
-    public void addLoanedBookTitle(Title title) {
-        loanedBooks.add(title);
+    public void addLoanedBook(Book book) {
+        loanedBooks.add(book);
+    }
+
+    public boolean returnLoan(Book book) {
```

Comment:
How does `isLoanReturnedSuccessfully` sound? Though actually now I think it is a terrible function to have - because it returns true if book was returned successfully, and false if book was not on loan. I'll separate this behaviour into another function

Reply:
Done

Reply:
`How does isLoanReturnedSuccessfully sound?`
That sounds alright.

`I can separate this behaviour into another function?`
I'm not in a position to comment but it seems alright as it currently is? But go with your instinct!